BANKI-answers.md

```diff
@@ -1451,10 +1451,42 @@ function factorial(num) {
 
 - [ ] What is polymorphism?
 
-  - **Explanation:**
-  - **Use:**
+  - **Explanation:** Polymorphism is a concept of Object Oriented Paradigm(OOP) that provides a way to perform a single action in different ways.
```

Comment:
I think this should be "Object-oriented programming (OOP) paradigm"

Reply:
I have made the suggested change. Thanks!